CenterForOpenScience/ember-osf-web
# [LUP][ENG-299]PATCH to API when resetting client secret for developer app.

## Purpose

Make a PATCH request to API endpoint to reset client secret, instead of using a custom POST.

## QA Notes

Resetting client secret should work as usual. Click `show client secret` before and after resetting to make sure it does change.
